src/components/forms/PlanForm/helpers.ts

```diff
@@ -757,6 +772,11 @@ export function getPlanFormValues(planObject: PlanDefinition): PlanFormFields {
       ? taskGenerationStatuses[2]
       : taskGenerationStatuses[1];
 
+  const teamAssignmentStatus: string =
+    teamAssignmentStatusContext.length > 0
+      ? taskGenerationStatusContext[0].valueCodableConcept
+      : '';
```

Comment:
1. Should this be `undefined`?
2. If this is not set (i.e. it is ''), what happens to the plan object?  Does it end up with this useContext value being ''?

Reply:
Am using `formValue.teamAssignmentStatus && formValue.teamAssignmentStatus.trim()` to not set for empty strings